index.js

```diff
@@ -399,10 +456,12 @@ ko.bindingHandlers.fastForEach = {
         $context: context
       });
     }
+    if (FastForEach.DEBUG)
```

Comment:
@cervengoc I was wondering that, too. :)

The easiest way I can think of, that sticks to the conformity without modifying FFE, is to pass an `afterAdd`.  The `afterAdd` handler is called with the instance as the `foreachInstance` property.

Another, cleaner option, may be to just instantiate FFE directly, instead of via `applyBindings -> fastForEach.init`.  In other words, call directly `ffe = new FastForEach({*params*})`.  We'd just be emulating the `init` call, so the only non-trivial thing you need is a fake KO context, which you can get with e.g.

```
var div = document.createElement(div)
ko.applyBindings(vm, div)
var $context = ko.contextFor(div)
var ffe = new FastForEach({  data: test_viewmodel, element: div, $context: context })
```

Just a thought. :)